content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -73,22 +75,26 @@ The following `Service` is used for DNS lookups between Cassandra Pods and clien
 1. Launch a terminal window in the directory you downloaded the manifest files.
 1. Create a `Service` to track all Cassandra StatefulSet Nodes from the `cassandra-service.yaml` file:
 
-      ```shell
-      kubectl create -f https://k8s.io/examples/application/cassandra/cassandra-service.yaml
-      ```
+    ```bash
```

Comment:
FYI `bash` and `shell` are just aliases to the same behavior from the lexer.

Reply:
Yeah, I've been seeing plenty of both. Might be a good idea to choose a standard.

Reply:
@mistyhacks Okay, `git grep` is telling me that `shell` is used 1794 times while `bash` is used 87 times. So the desire lines say `shell`.